src/lib.rs

```diff
@@ -130,7 +131,7 @@ impl<T, const N: usize> AlignedStorage<T, N> {
     fn new() -> Self {
         Self {
             _align: [],
-            storage: MaybeUninit::uninit_array(),
+            storage: unsafe { MaybeUninit::uninit().assume_init() },
```

Comment:
This is most certainly UB, please confirm your PR with MIRI before submitting an update.